l10n_ro_partner_unique/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+
+from openerp import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    sql_constraints = [
+        ('vat_nrc_uniq',
+         'unique (id)',
```

Comment:
@moylop260 it is a placeholder for the constraint created in _auto_init. But @feketemihai, I believe this is not necessary, especially because the member is usually called \_sql_constraints with an underscore ;-). Maybe best to remove it?